src/app/shared/date/timepicker.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
```

Comment:
This time picker could probably also be implemented as an add-on to the datepicker. We could also use the pikaday fork that adds time picker support, not sure if it is well maintained though.